registry.json

```diff
@@ -3372,5 +3372,20 @@
     "telegram": "DeCentN2Madness",
     "operatorName": "Jeff Hales",
     "github": "qbass"
+  },
+  {
+    "ticker": "USONE",
+    "poolId": "576e0390ac54e3751afdaa8aa256f41b84a6324006f23991f6fdefb7",
+    "social": {
+      "facebook": "",
+      "twitter": "as_usone",
+      "website": "https://usas.one",
+      "reddit": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "us_as_one",
```

Comment:
please use a personal telegram handle. No groups or pages

Reply:
.

Reply:
Personal telegram is @SJPMurphy